src/platform/mhv/secondary-nav/tests/MhvSecondaryNavItem.unit.spec.jsx

```diff
@@ -98,7 +127,7 @@ describe('MHV Secondary Navigation Item Component', () => {
 
   describe('reports custom events to GA', async () => {
     it('when a link is clicked', async () => {
-      const href = '/my-health/unit-test';
+      const href = '#/my-health/unit-test';
```

Comment:
Note: There was an error being thrown to the console. Adding `#` is a workaround, so there is no navigation incurred, but the event is still fired.
`Error: Not implemented: navigation (except hash changes)`
Ref: https://stackoverflow.com/a/77617601/4334721